### PR TITLE
DOC-7500: Invite People page needs animated GIFs for mobile app workflows

### LINKED ISSUE #7500: Invite People page needs animated GIFs for mobile app workflows
## Context

Add visuals and short animated GIFs on the Invite People page to help visual learners learn how to perform tasks using the Mattermost mobile app.

## Docs Request

- Add appropriate visuals/GIFs for the `Send a direct invite` section under the mobile app tab.
- Add appropriate visuals/GIFs for the `Send a team invite link` section under the mobile app tab.

## Additional Notes

- Use the latest Mattermost mobile app and the standard Mattermost Denim theme to capture all workflows.
- Avoid drawing lines, arrows, boxes, or callouts on images.
- Don't forget to include ALT Text descriptions for all new image assets.
- Use existing docs pages with images as a starting point for code guidance. The Docs team can help provide RST syntax support for publication.
- Save static image files in PNG format within the docs repository's `/images` directory.

### PR DESCRIPTION
**Pull request by Rutam21**
### Summary

This PR adds visual workflows for the Mattermost mobile app tabs on the [Invite People](https://docs.mattermost.com/collaborate/invite-people.html#) page

### Ticket Link

Fixes #7500

- [x] This PR follows Mattermost Contributing Guidelines.

### CODE REVIEW

**Comment by Rutam21** [comment]
@cwarnermm Please review and suggest changes, if any. Thanks.

File: source/collaborate/invite-people.rst [hunk 1]
```diff
@@ -48,7 +48,18 @@ Direct invites are invitation emails sent from your team's server directly to th
 .. tab:: Mobile
 
   1. Tap the |plus| icon in the top right corner of the screen.
+
+    .. image:: ../images/direct-mobile-invite-options.jpg
```
Comment (cwarnermm) [inline source/collaborate/invite-people.rst:52]: @Rutam21 - Please scale the new images by adding ``:scale: 60`` on the line after ``:alt:``. Let's start with scaling to 60% and see how they display on the page.

File: source/collaborate/invite-people.rst [hunk 2]
```diff
@@ -48,7 +48,18 @@ Direct invites are invitation emails sent from your team's server directly to th
...
+      :alt: When you click on the + button, you can access more options from the popup window at bottom.
```
Comment (cwarnermm) [inline source/collaborate/invite-people.rst:53]:
```suggestion
      :alt: When you select +, you can access more options from the popup window.
```

File: source/collaborate/invite-people.rst [hunk 3]
```diff
@@ -48,7 +48,18 @@ Direct invites are invitation emails sent from your team's server directly to th
...
+
   2. Tap the **Invite people to the team** option.
+
+    .. image:: ../images/direct-mobile-invite-by-email.jpg
+      :alt: You can invite an user using their email address.
```
Comment (cwarnermm) [inline source/collaborate/invite-people.rst:58]:
```suggestion
      :alt: You can invite a user using their email address.
```

File: source/collaborate/invite-people.rst [hunk 4]
```diff
@@ -48,7 +48,18 @@ Direct invites are invitation emails sent from your team's server directly to th
...
+
+    .. image:: ../images/direct-mobile-invite-by-name.jpg
+      :alt: You can invite an user using their name.
```
Comment (cwarnermm) [inline source/collaborate/invite-people.rst:61]:
```suggestion
      :alt: You can invite a user using their name.
```

File: source/collaborate/invite-people.rst [hunk 5]
```diff
@@ -72,8 +83,20 @@ You can share a unique URL that takes users to a Mattermost account creation pag
 .. tab:: Mobile
 
   1. Tap the |plus| icon in the top right corner of the screen.
+
+    .. image:: ../images/direct-mobile-invite-options.jpg
+      :alt: When you click on the + button, you can access more options from the popup window at bottom.
```
Comment (cwarnermm) [inline source/collaborate/invite-people.rst:88]:
```suggestion
      :alt: When you select +, you can access more options from the popup window.
```

File: source/collaborate/invite-people.rst [hunk 6]
```diff
@@ -72,8 +83,20 @@ You can share a unique URL that takes users to a Mattermost account creation pag
...
+
   2. Tap the **Invite people to the team** option.
+
+    .. image:: ../images/direct-mobile-invite.jpg
+      :alt: You can also invite an user by clicking on the Share link button.
```
Comment (cwarnermm) [inline source/collaborate/invite-people.rst:93]:
```suggestion
      :alt: You can also invite a user by selecting Share link.
```

File: source/collaborate/invite-people.rst [hunk 7]
```diff
@@ -72,8 +83,20 @@ You can share a unique URL that takes users to a Mattermost account creation pag
...
+
   3. Tap **Share link**.
+
+    .. image:: ../images/direct-mobile-invite-by-share-link.jpg
+      :alt: You can invite an user by sharing the invite link directly.
```
Comment (cwarnermm) [inline source/collaborate/invite-people.rst:98]:
```suggestion
      :alt: You can invite a user by sharing the invite link directly with them.
```

**Review by cwarnermm — CHANGES_REQUESTED** [review]

File: source/collaborate/invite-people.rst (hunk 1 above)
Comment (Rutam21) [reply source/collaborate/invite-people.rst:52]: This fix is applied to all images in this PR.

**Comment by Rutam21** [comment]
@cwarnermm All the requested changes have been done. Please review. Thanks.

File: source/collaborate/invite-people.rst [hunk 8]
```diff
@@ -48,7 +48,21 @@ Direct invites are invitation emails sent from your team's server directly to th
 .. tab:: Mobile
 
   1. Tap the |plus| icon in the top right corner of the screen.
+
+    .. image:: ../images/direct-mobile-invite-options.jpg
+      :alt: When you select +, you can access more options from the popup window.
+      :scale: 60
```
Comment (cwarnermm) [inline source/collaborate/invite-people.rst:54]:
```suggestion
      :scale: 40
```

File: source/collaborate/invite-people.rst [hunk 9]
```diff
@@ -48,7 +48,21 @@ Direct invites are invitation emails sent from your team's server directly to th
...
+
+    .. image:: ../images/direct-mobile-invite-options.jpg
+      :alt: When you select +, you can access more options from the popup window.
+      :scale: 30
+
   2. Tap the **Invite people to the team** option.
+
+    .. image:: ../images/direct-mobile-invite-by-email.jpg
+      :alt: You can invite a user using their email address.
+      :scale: 60
```
Comment (cwarnermm) [inline source/collaborate/invite-people.rst:60]:
```suggestion
      :scale: 30
```

File: source/collaborate/invite-people.rst [hunk 10]
```diff
@@ -48,7 +48,21 @@ Direct invites are invitation emails sent from your team's server directly to th
...
+
+    .. image:: ../images/direct-mobile-invite-by-name.jpg
+      :alt: You can invite a user using their name.
+      :scale: 60
```
Comment (cwarnermm) [inline source/collaborate/invite-people.rst:64]:
```suggestion
      :scale: 30
```

File: source/collaborate/invite-people.rst [hunk 11]
```diff
@@ -72,8 +86,23 @@ You can share a unique URL that takes users to a Mattermost account creation pag
 .. tab:: Mobile
 
   1. Tap the |plus| icon in the top right corner of the screen.
+
+    .. image:: ../images/direct-mobile-invite-options.jpg
+      :alt: When you select +, you can access more options from the popup window.
+      :scale: 60
```
Comment (cwarnermm) [inline source/collaborate/invite-people.rst:92]:
```suggestion
      :scale: 30
```

File: source/collaborate/invite-people.rst [hunk 12]
```diff
@@ -72,8 +86,23 @@ You can share a unique URL that takes users to a Mattermost account creation pag
...
+
   2. Tap the **Invite people to the team** option.
+
+    .. image:: ../images/direct-mobile-invite.jpg
+      :alt: You can also invite a user by selecting Share link button.
+      :scale: 60
```
Comment (cwarnermm) [inline source/collaborate/invite-people.rst:98]:
```suggestion
      :scale: 30
```

File: source/collaborate/invite-people.rst [hunk 13]
```diff
@@ -72,8 +86,23 @@ You can share a unique URL that takes users to a Mattermost account creation pag
...
+
   3. Tap **Share link**.
+
+    .. image:: ../images/direct-mobile-invite-by-share-link.jpg
+      :alt: You can invite a user by sharing the invite link directly with them.
+      :scale: 60
```
Comment (cwarnermm) [inline source/collaborate/invite-people.rst:104]:
```suggestion
      :scale: 30
```

**Comment by varghesejose2020** [comment]
@lindy65 please assign me

**Comment by varghesejose2020** [comment]
The changes in this PR have been tested in the preview environment and changes are working as expected. Adding screenshots for confirmation
> Send a direct invite

IMG_20241022_215841
> Send a Team invite

IMG_20241022_215809
cc: @lindy65

**Comment by lindy65** [comment]
Thanks for testing @varghesejose2020 - counted towards Hacktoberfest

cc @cwarnermm

**Pull request merged**